gym_cap/envs/agent.py

```diff
@@ -229,6 +230,62 @@ def get_loc(self):
     def report_loc(self):
         print("report: position x:%d, y:%d" % (self.x, self.y))
 
+    def get_obs(self, com_ground, com_air, env, *args):
+        if self.team == BLUE:
+            myTeam = env.get_team_blue()
+        else:
+            myTeam = env.get_team_red()
+
+        a = 39              # env.map_size[0]*2-1
+        b = 39              # env.map_size[1]*2-1
+        obs = np.full(shape = (a, b), fill_value = OBSTACLE)
```

Comment:
Assign 'UNKNOWN' instead of OBSTACLE for out of vision. If the place is 'out of boarder,' use OBSTACLE.